src/components/ProfileMisison.js

```diff
@@ -0,0 +1,29 @@
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { leaveMission } from '../redux/missions/missionSlice';
+import './styles/profile.css';
+
+export default function ProfileMission({ mission }) {
+  const dispatch = useDispatch();
+
+  return (
+    <div>
+      {mission.reserved ? (
+        <div key={mission.mission_id} className="details">
+          <div className="Mision_name">{mission.mission_name}</div>
+          <button type="button" className="button-active" onClick={() => dispatch(leaveMission(mission.mission_id))}>Leave Mission</button>
+        </div>
+      ) : (
+        ''
```

Comment:
The same applies here for your my profile page I strongly suggest you style this according to design guidelines:

This is your design:
![Screenshot 2023-04-14 at 8 04 20 PM](https://user-images.githubusercontent.com/79506105/232124649-27944b2e-44fa-46e1-8a91-8aafbb8e68fe.png)


Please stick to this design:


![My Profile](https://user-images.githubusercontent.com/79506105/232124780-99e3ddb4-6ed9-462f-a887-c25ee6436761.png)